common/src/main/java/io/netty/util/DomainNameMapping.java

```diff
@@ -141,4 +136,23 @@ public V map(String input) {
     public String toString() {
         return StringUtil.simpleClassName(this) + "(default: " + defaultValue + ", map: " + map + ')';
     }
+
+    /**
+     * Creates a new builder for {@link DomainNameMapping}.
+     *
+     * @param defaultValue the default value for {@link #map(String)} to return when nothing matches the input
+     */
+    public static <V> DomainMappingBuilder<V> withDefaultValue(V defaultValue) {
+        return withInitialCapacityAndDefaultValue(4, defaultValue);
+    }
+
+    /**
+     * Creates a new builder for {@link DomainNameMapping}.
+     *
+     * @param initialCapacity initial capacity for the internal map
+     * @param defaultValue    the default value for {@link #map(String)} to return when nothing matches the input
+     */
+    public static <V> DomainMappingBuilder<V> withInitialCapacityAndDefaultValue(int initialCapacity, V defaultValue) {
+        return new DomainMappingBuilder<V>(initialCapacity, defaultValue);
+    }
```

Comment:
Actually, could you remove these two methods because it overlaps with the public constructors of the builder class?